### DISCUSSION TITLE
Transaction Historial missing info

### DISCUSSION THREAD

**Discussion by arcturiano** [BitCannaGlobal/WeedWallet#84] [General]
1. Submit proposal transactions are missing information like other transactions have:

From address
To address
Amount (may not be necessary here)

2. Submit Proposal transactions also seem to have an icon missing.  
![TransactionHistoricalMissingInfo](https://github.com/BitCannaGlobal/WeedWallet/assets/94879750/a2e8335f-deae-4415-93e1-6f0de2f24e6f)

**Comment by TJ-BitCanna** [comment]
Ah yes. 
To address might not be relevant, but the from address and the amount are. As well as the pictogram.

We'll take it with us.

**Comment by TJ-BitCanna** [comment]
Suggested issue:
- complete details for the "Submit proposal" transaction type on the Transactions page
- check if this also accounts for "Deposit funds for proposal" & "vote" transactions

**Comment by TJ-BitCanna** [comment]
Completed :)
